draft-ietf-httpbis-replay.md

```diff
@@ -251,9 +251,11 @@ Early-Data: 1
 
 ~~~
 
-An intermediary that forwards a request received in TLS early data MUST send it
-with the `Early-Data` header field set to "1" (i.e., it adds it if not present
-in the request).
+An intermediary that forwards a request prior to the completion of the TLS
+handshake MUST send it with the `Early-Data` header field set to "1" (i.e., it
+adds it if not present in the request).  This is also true if the intermediary
```

Comment:
s/true/required/? But if that's the case, it should really be a REQUIREMENT.

Reply:
Sure. Reworded.